cmd/containerboot/kube.go

```diff
@@ -81,7 +81,7 @@ func initKubeClient(root string) {
 		kubeclient.SetRootPathForTesting(root)
 	}
 	var err error
-	kc, err = kubeclient.New()
+	kc, err = kubeclient.New("tailscale-container-client")
```

Comment:
Client now has a name, so that when Events are emitted, we can write emitter's name on the Event (i.e kube store client or containerboot client)